modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -110,49 +109,49 @@ def number_and_street
     end
 
     def city
-      veteran.dig('address', 'city') || ''
+      veteran_data.dig('address', 'city') || ''
```

Comment:
renamed from veteran to veteran_data so no conflict with new veteran method above, which creates an Appellant.